test/module/options/normalizeUrlSpec.js

```diff
@@ -0,0 +1,26 @@
+import 'source-map-support/register';
```

Comment:
Do we have to import this file? I can't really remember if it's needed for all files